packages/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_kibana_client.ts

```diff
@@ -80,5 +83,52 @@ export class ApmSynthtraceKibanaClient {
     }
 
     this.logger.info(`Installed APM package ${packageVersion}`);
+    return packageVersion;
```

Comment:
I wonder if it's worth mixing this into all the side-effects that this method has just to make the argument optional.

Reply:
My thinking was since we are always (as of yet) only interested in the latest version, including the request to get the version in the `installApmPackage` function was more convenient while leaving an option to pass in a specific version if someone is needing to do that.  In the case where we just want the latest version, I thought it would be useful to return the version installed since we didn't specify one.  And then since the `ApmSynthtraceEsClient` needs the version in order to instantiate it, it was also convenient.  I could also just call `fetchLatestApmPackageVersion` from the kibana client and pass that into the `ApmSynthtraceEsClient`.